src/pages/Admin/Content/ContentForm.jsx

```diff
@@ -1,84 +1,88 @@
-import React, { useEffect, useState } from "react";
-import { CoverGreetings } from "../../../components/Utils/Utils";
-import { useLocation, useParams } from "react-router-dom";
+import React, { useEffect, useState } from 'react'
+import { CoverGreetings } from '../../../components/Utils/Utils'
+import { useLocation, useParams } from 'react-router-dom'
+import { Input } from '../../../components/Input'
+import { Button, colorSchema } from '../../../components/Button'
+import { Label } from '../../../components/Label'
+import { Select } from '../../../components/Select'
 
 const ContentForm = () => {
-  const location = useLocation();
-  const { id } = useParams();
-  const [inputError, setInputError] = useState({ error: false, message: "" });
+  const location = useLocation()
+  const { id } = useParams()
+  const [inputError, setInputError] = useState({ error: false, message: '' })
   const [informativeMessages, setInformativeMessages] = useState({
-    greetings: "",
-    btnSubmitMessage: "",
-  });
+    greetings: '',
+    btnSubmitMessage: '',
+  })
 
   const [inputs, setInputs] = useState({
-    title: "",
-    description: "",
-    link: "",
-    temario: "",
-    level: "",
-    importance: "",
-  });
+    title: '',
+    description: '',
+    link: '',
+    temario: '',
+    level: '',
+    importance: 0,
+  })
```

Comment:
a single suggestion

if you want you can create another file like `ContentForm.constants.js` and save default data that you want put in a any `useState`